src/lib/streamlines/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum LayerType {
+  Streamline = 'streamline',
+  Static = 'static',
+}
```

Comment:
If this is a type, shouldn't we do:
```suggestion
export const LayerTypeEnum = {
  Streamline: 'streamline',
  Static: 'static',
} as const

export type LayerType = (typeof LayerTypeEnum)[keyof typeof LayerTypeEnum]
```

Reply:
Renamed to `LayerKind`